src/core/utilities.js

```diff
@@ -99,12 +99,12 @@ function objectType( obj ) {
 		case "Date":
 		case "RegExp":
 		case "Function":
+		case "Symbol":
 			return type.toLowerCase();
 	}
 	if ( typeof obj === "object" ) {
 		return "object";
 	}
```

Comment:
I can't figure any other type than "object" here, maybe I should remove this if and return `"object"` at this part as well.

This is remaining as a fail safe only.